src/Compilers/Core/CodeAnalysisTest/Text/EncodedStringTextTests.cs

```diff
@@ -97,7 +97,7 @@ public void CheckSum_SHA256()
         public void Decode_NonUtf8()
         {
             var utf8 = new UTF8Encoding(false, true);
-            var encoding = Encoding.GetEncoding(name: "Latin1");
+            var encoding = Encoding.Default;
```

Comment:
Can we change this line to `Encoding.GetEncoding(codePage: 1252)`?  

Soon we are going to begin running our unit test on top of CoreCLR at which point this line won't compile anymore.  The `GetEncoding` call will compile but fail at runtime.  That is okay though because we can then mark the test to only run on Desktop.